kani-compiler/src/codegen_cprover_gotoc/codegen/assert.rs

```diff
@@ -66,6 +71,7 @@ impl<'tcx> GotocCtx<'tcx> {
         )
     }
 
+    /// A shorthand for generating a CBMC assert-false. TODO: This should probably be eliminated!
```

Comment:
Do we want an issue for this?

Reply:
It's the assertions audit. I will be doing it next. :)